docker/spark/Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN wget -q http://central.maven.org/maven2/net/logstash/log4j/jsonevent-layout/
 RUN wget -q http://central.maven.org/maven2/net/minidev/json-smart/1.1.1/json-smart-1.1.1.jar -P /spark/jars/
 RUN wget -q http://central.maven.org/maven2/org/apache/hadoop/hadoop-aws/2.7.3/hadoop-aws-2.7.3.jar -P /spark/jars/
 RUN wget -q http://central.maven.org/maven2/com/amazonaws/aws-java-sdk/1.7.4/aws-java-sdk-1.7.4.jar  -P /spark/jars/
+RUN wget -q https://yotpo-public.s3.amazonaws.com/spark-atlas-connector/spark-atlas-connector-assembly.jar  -P /spark/jars/
```

Comment:
Get it from here:
https://repo.hortonworks.com/content/repositories/releases/com/hortonworks/spark/spark-atlas-connector-assembly/0.1.0.3.1.2.1-1/spark-atlas-connector-assembly-0.1.0.3.1.2.1-1.jar

Add version variable above perhaps

Reply:
Maybe we should let people download this with --jars instead?

Reply:
im trying to do this : `spark-shell --jars https://repo.hortonworks.com/content/repositories/releases/com/hortonworks/spark/spark-atlas-connector-assembly/0.1.0.3.1.4.9-1/spark-atlas-connector-assembly-0.1.0.3.1.4.9-1.jar --conf spark.sql.hive.convertMetastoreParquet=false  --conf spark.serializer=org.apache.spark.serializer.KryoSerializer  --conf spark.sql.catalogImplementation=hive  --conf spark.hadoop.hive.metastore.uris=thrift://172.16.123.1:9083  --conf spark.sql.warehouse.dir=/warehouse --conf spark.extraListeners=com.hortonworks.spark.atlas.SparkAtlasEventTracker  --conf spark.sql.queryExecutionListeners=com.hortonworks.spark.atlas.SparkAtlasEventTracker  --conf spark.sql.streaming.streamingQueryListeners=com.hortonworks.spark.atlas.SparkAtlasStreamingQueryEventTracker` 
and its now working for me